scripts/policy/frameworks/cluster/websocket/server.zeek

```diff
@@ -0,0 +1,54 @@
+##! Script to load for running a single WebSocket server.
+##!
+##! This script is mostly meant for ad-hoc testing. In a Zeekctl environment,
+##! the ``UseWebSocket`` option should be used instead.
+##!
+##! Note that if :zeek:see:`Cluster::backend` is ``CLUSTER_BACKEND_NONE`` at the
+##! time this script is loaded, it loads the ZeroMQ cluster backend and starts a
+##! locally running XPUB/XSUB proxy thread. If you instead want to use Broker's hub
+##! functionality instead, ensure to load policy/frameworks/cluster/backend/broker
+##! before loading this script.
+##!
+##! Note also that this script will raise a fatal error if the cluster backend
+##! is :zeek:see:`Cluster::CLUSTER_BACKEND_NONE`, but :zeek:see:`Cluster::nodes`
+##! is populated with entries.
```

Comment:
Are we sure this is the right place to make that call? The reasoning makes sense to me, but I wonder why we would error on it when the user wants to enable the webserver?